src/test/java/teammates/test/cases/ui/browsertests/InstructorHomePageUiTest.java

```diff
@@ -250,7 +250,8 @@ public void testRemindActions(){
         homePage.goToPreviousPage(InstructorHomePage.class);
         
         ______TS("remind action: OPEN feedback session - inner button");
-        
+
+        homePage.waitForAjaxLoaderGifToDisappear();
```

Comment:
Considering the tests already take so long, I think its better to wait only when necessary?
With more ajax features being added, we can't just do a wait all the time.